app/assets/javascripts/modules/DataRequestActions.js

```diff
@@ -18,8 +20,25 @@ moj.Modules.DataRequestActions = {
         $mm_field.val(month);
         $yyyy_field.val(year);
       }
-    })
+    });
+
+    $('#new_data_request').submit(function() {
+      self.clearNoteField();
+    });
+
+    $('#edit_data_request').submit(function() {
+      self.clearNoteField();
+    });
 
   },
 
+  clearNoteField: function() {
+    if (!$('#data_request_request_type_other').is(':checked')) {
```

Comment:
You could reduce some of the logic here by passing the ID selectors and calling passing them at lines 26 and 30 - something like:

```javascript
self.clearNoteField('#data_request_type_other', '#request_type_other_panel')

// then as a function:

clearNoteFeild: function(request_type, panel){
   if(!$(request_type).is(':checked')) {
    $(panel).empty()
   }
}
```

Small thing though really :)